Source/santad/SNTSyncdQueue.mm

```diff
@@ -185,6 +195,20 @@ - (void)dispatchBlockOnSyncdQueue:(void (^)(void))block {
   });
 }
 
+- (void)exportTelemetryFile:(NSFileHandle *)telemetryFile
+          completionHandler:(void (^)(BOOL))completionHandler {
+  LOGE(@"Syncd queue, about to dispatch...");
```

Comment:
Leftover? Maybe make LOGD if you want to keep?

Reply:
Ope, leftover